WordPressMocks/src/main/assets/mocks/mappings/wpcom/activity/wpcom_v2_site_activity.json

```diff
@@ -38,7 +38,7 @@
                             "name": "Jetpack"
                         },
                         "type": "Announce",
-                        "published": "2021-05-06T06:23:10.798+00:00",
+                        "published": "{{now format='yyyy-MM-dd'}}T{{now format='HH:mm:ss.SSSZ'}}",
```

Comment:
I think you should use our custom `{{fnow}}` tag instead of `{{now}}` to avoid the issues we had in the past with dates being localized by the `now` tag – see pbMoDN-13F-p2 and https://github.com/wordpress-mobile/WordPress-Android/pull/12831

(side note: we should maybe consider adding a linter rule for that to ensure all our mocks use `fnow` instead of `now`, as I'm sure many people are not aware of this glitch we had back then…)

Reply:
> we should maybe consider adding a linter rule for that to ensure all our mocks use fnow instead of now, as I'm sure many people are not aware of this glitch we had back then…

This would be gold!

Reply:
@AliSoftware Thanks for suggestion about `{{fnow}}`! 

I'm trying to wrap my head around how we'd implement a similar custom handlebar extension in this repo, so we can use it for the WPiOS UI tests... Let me know if this sounds right:

- Add a new `UnlocalizedDateHelper.java` file  in `src/main/java/org/wordpress/ios/mocks` and copy the code from [here](https://github.com/AliSoftware/WordPress-Android/blob/5449f314d077dc43dc4eb78dd1eb9e3ad61eae47/WordPress/src/androidTest/java/org/wordpress/android/support/BaseTest.java#L110-L173)
- Register the custom extension in [scripts/start.sh](https://github.com/wordpress-mobile/WordPressMocks/blob/97068093636de1fe6b8378c5b64640806c591da8/scripts/start.sh#L20-L23) by specifiying `--extensions org.wordpress.ios.mocks.UnlocalizedDateHelper` (Ref: http://wiremock.org/docs/running-standalone/)

I may not be able to figure this out before the JP release, in which case I'll create a new issue.

Reply:
@momo-ozawa this seems like the right plan 👍 

Indeed since for iOS we run WireMock standalone, the only solution to register the fnow extension seems to be via `start.sh`.

As for where to declare the extension, not sure it really matters, but my reasoning would be that this handlebar extension doesn't really have to be restricted to iOS and would make sense to be used for any platform (even if, iinm, for Android I think we don't use this repo anymore and only use those for iOS?? So, granted in practice that might be the only platform for which we'll use this, but logically speaking that extension is not really related to an iOS-only feature, so maybe something more like `src/main/java/org/wordpress/common/mocks`? 🤷

Reply:
> This would be gold!

@mokagio I don't think we have a linter for that, so I think we'd need to implement some manual `grep`-ing for that (e.g. grep for `{{ *now `) right? Not sure if Peril would be worth for that vs just a CI run script (though we might want to share that linting between the mocks hosted within WordPressAndroid repo and the ones in this repo 🤔 )